book/src/stake-delegation-and-rewards.md

```diff
@@ -0,0 +1,127 @@
+# Stake Delegation and Reward
+
+This proposal covers the current architecture and proposed changes to how stakes
+are delegated to validators.  This design focuses on the software architecture
+for the actual on-chain programs.  Incentives for staking is covered in [staking
+rewardsd](staking-rewards.md).
+
+The main problem this proposal is solving is to allow many delegated stakes to
```

Comment:
I'd love to see lots of effort into making this statement crystal clear. Until we're all on the same page about the problem statement, all conversation following this problem statement seems wasteful to me.

Using current terminology, "many delegated stakes" translates to "many delegates".  By "generate rewards with a single validator vote", do you mean "redeem credits after voting"?  Currently, you don't need permission from the validator to redeem credits, so not clear yet what problem your solving.

Reply:
“generate rewards with a single validator vote without permission from the
validator.”

The current design requires a validator to submit a different vote for each stake. Therefore the validator can censor stakes delegated to it.   And the number of votes is equal to the number of stakers, and not the number of validators.